dataset/src/main/java/org/kitesdk/examples/data/BaseEventsTool.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2014 Cloudera, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kitesdk.examples.data;
+
+import java.util.LinkedList;
+import java.util.List;
+import org.apache.hadoop.conf.Configured;
+import org.apache.hadoop.util.Tool;
+
+public abstract class BaseEventsTool extends Configured implements Tool {
```

Comment:
It doesn't look like any of the code in this class is used, so it would be better to remove it and make GenerateEvents implement Tool directly.